pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix

```diff
@@ -1,17 +1,16 @@
-{ fetchurl, bitwig-studio1, pulseaudio, xorg }:
+{ stdenv, fetchurl, bitwig-studio2, xorg, ... }:
```

Comment:
Why the swap to `...` var args?  And it doesn't look like `stdenv` is used here?

Reply:
true, will change that.